test/roby_app/spawn_server/test_spawn_server.rb

```diff
@@ -35,13 +35,14 @@ def upload_log(host, port, certificate, user, password, localfile)
             ftp.login(user, password)
             lf = File.open(localfile)
             ftp.storbinary("STOR #{File.basename(localfile)}", lf, Net::FTP::DEFAULT_BLOCKSIZE)
+            lf.close
```

Comment:
In this particular case,

~~~
File.open(localfile) do |lf|
  ftp.storbinary("STOR #{File.basename(localfile)}", lf, Net::FTP::DEFAULT_BLOCKSIZE)
end
~~~

is much better, as the file will be closed even if `storbinary` throws an exception